### ISSUE TITLE
GPSR product placed on the market before 13.12.

### ISSUE THREAD

**Issue by dominikahr** [allegro/allegro-api#10516]
Hi, 

in a support article for sellers I noticed, that there is a new parameter in the listing form, where the seller can select, if the product was placed on the EU market before 13.12. Is there somewhere more information about this parameter? Can we also send this via API? 

And if the user will select, that the product was placed on the EU market before 13.12., does that mean that the merchant does not have to provide the safety information and responsible producer? 

![image](https://github.com/user-attachments/assets/837470c7-86fc-425b-8e43-29cfa094e871)

Thanks in advance! 

All the best, 

Dominika

**Comment by PrzemyslawLukanowski** [comment]
Yes, it can be sent via API, it is a temporary offer parameter - ID 250553. Temporary, because soon we will introduce a separate flag on the product level when creating an offer, in which you will be able to indicate whether the product was introduced to the EU market before 13.12.2024 or not. Yes, by filling it a merchant does not need to provide the safety information and responsible producer.

**Comment by dominikahr** [comment]
@PrzemyslawLukanowski thank you! Do you have any estimation when the new flag on the product level will be introduced? If it makes sense for us to also implement this temporary solution or if we should wait for the new one.

**Comment by PrzemyslawLukanowski** [comment]
We plan to introduce it by the end of this week.
